Project 1/shortstoryalt.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
The folder for this project needs to be renamed -- to not include spaces or capital letters, e.g. `project1/` not `Project 1/`